addons/hr_contract/models/resource.py

```diff
@@ -10,6 +10,7 @@ class ResourceCalendar(models.Model):
     _inherit = 'resource.calendar'
 
     contracts_count = fields.Integer("# Contracts using it", compute='_compute_contracts_count', groups="hr_contract.group_hr_contract_manager")
+    company_country_id = fields.Many2one('res.country', string='Country', related='company_id.partner_id.country_id')
```

Comment:
actually, this can be so as well, maybe cleaner

```suggestion
    company_country_id = fields.Many2one('res.country', string='Country', related='company_id.country_id')
```

Reply:
`country_id` on company is not searchable.

Reply:
okok, we can leave it like this then